pkg/flowkit/config/json/network.go

```diff
@@ -28,60 +29,102 @@ type jsonNetworks map[string]jsonNetwork
 
 // transformToConfig transforms json structures to config structure.
 func (j jsonNetworks) transformToConfig() (config.Networks, error) {
-	var networks config.Networks
+	networks := make(config.Networks, 0)
 
 	for networkName, n := range j {
-		network := config.Network{
-			Name: networkName,
-			Host: n.Host,
+		if n.Advanced.NetworkKey != "" && n.Advanced.Host != "" {
```

Comment:
if an advanced schema is passed it should contain both values else it should be a simple format.

Reply:
User can possibly set advanced.network key but not advanced.host.

Reply:
but what would then the host value be? that is the case where we need to show error right? since they are passing only the key but we don't know where we should connect

Reply:
Sounds good I will require advanced config to be complete

Reply:
https://github.com/onflow/flow-cli/pull/449/commits/b94433142876497e2697c8cb40ecc8b135bfaa70